tox.ini

```diff
@@ -41,7 +41,7 @@ basepython = python3
 commands_pre =
     mkdir -p {toxinidir}/parts/flake8
 commands =
-    isort --diff {toxinidir}/src setup.py
```

Comment:
Please do not remove this flag. It is very helpful.

Reply:
Just for clarity - the `diff` flag does three things:
- isort will always return an return code of 0 - no matter how many imports are wrongly sorted
- isort will no longer sort the imports automatically
- isort will show the diff what it would do to the console

This basically means, either we have to fix the import order manually according to the console output, or we have to temporarily remove the `diff` flag from the `isort` to let `isort` do the work.

Anyway, here we go - I restored the `diff` flag.

Reply:
Very well! Flag restored (comment see https://github.com/zopefoundation/DocumentTemplate/pull/56#discussion_r501480769 )

Reply:
`isort` should never run during tests in a way that changes code. It must retain the `--check-only` flag. Changing code should always remain an explicit action by a developer, not a side effect of running tests! I overlooked your removal, my bad, but I wonder why you introduce behavior changes to begin with. Just make the minimum changes required. Don't change test run behavior.

Reply:
The combination of `diff` and `check-only` is pretty surprising.

**only diff**
- shows diff
- does not change code
- return code 0

**diff and checkonly**
- shows diff
- does not change code
- return code != 0 => test run fails

Reply:
restored `check-only` flag which will make test runs fail when imports are not sorted properly :+1: